jetty-util/src/test/java/org/eclipse/jetty/util/UrlEncodedUtf8Test.java

```diff
@@ -35,6 +35,29 @@
 
     static final Logger LOG = Log.getLogger(UrlEncodedUtf8Test.class);
 
+    @Test
+    public void testIncompleteSequestAtTheBeginning() throws Exception
```

Comment:
We have this test on the websocket code tree already.
It's mandatory to throw, never replace, and close the connection.